SpeechService/SpeechService.cs

```diff
@@ -364,12 +366,19 @@ private void speak(MemoryStream stream, string voice, string speech)
                     // Keep XML version at 1.0. Version 1.1 is not recommended for general use. https://en.wikipedia.org/wiki/XML#Versions
                     if (speech.Contains("<"))
                     {
+                        SpeakFn speakSsml = synth.SpeakSsml;
+                        if (voice.StartsWith("CereVoice "))
+                        {
+                            Logging.Debug("Working around CereVoice SSML support");
+                            speakSsml = synth.Speak;
+                        }
+
                         Logging.Debug("Obtaining best guess culture");
                         string culture = @" xml:lang=""" + bestGuessCulture() + @"""";
                         Logging.Debug("Best guess culture is " + culture);
                         speech = @"<?xml version=""1.0"" encoding=""UTF-8""?><speak version=""1.0"" xmlns=""http://www.w3.org/2001/10/synthesis""" + culture + ">" + escapeSsml(speech) + @"</speak>";
                         Logging.Debug("Feeding SSML to synthesizer: " + escapeSsml(speech));
-                        synth.SpeakSsml(speech);
+                        speakSsml(speech);
```

Comment:
I'm not sure `SpeakFn` is needed and I think it may make the code a bit harder to follow. Why not do something like this?
```cs
if (voice.StartsWith("CereVoice "))
{
    // Cereproc voices do not respect `SpeekSSML`, but they do handle SSML via the `Speak` method.
    Logging.Debug("Working around CereVoice SSML support");
    synth.Speak(speech);
}
else
{
    synth.SpeakSSML(speech);						
}
```

Reply:
Fair enough.

Reply:
In fact I have just rechecked this with the latest version of Cereproc Heather and it would appear this workaround is no longer needed.

If someone could confirm this with any other Cereproc voices they own, we could even drop this workaround entirely.

Reply:
I just tested using the phrase `This is how I will sound in your <phoneme alphabet="ipa" ph="kəʊbrə">Cobra</phoneme> <phoneme alphabet="ipa" ph="mɑːk">Mk</phoneme> <phoneme alphabet="ipa" ph="θri">III</phoneme>`. It looks to me like this workaround is still valid (IPA is used when `synth.Speak()` is invoked but it is not used when `synth.SpeakSsml()` is invoked).

Reply:
Superseded by #1620.